app/Repositories/SyllableRepository.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories;
+
+use App\Entities\SelectedSyllable;
+use App\Entities\Syllable;
+use PDO;
+
+class SyllableRepository
+{
+    public function __construct(
+        private readonly PDO $connection
+    ){
+    }
+
+    public function loadSyllablesFromFileToDb(string $fileName): void
+    {
+        $path = dirname(__DIR__) . $fileName;
+
+        if (!file_exists($path)) {
+            throw new \InvalidArgumentException("File not found with path {$path} and file name {$fileName}!");
+        }
+
+        try {
+            $stmt = $this->connection->prepare(
+                "LOAD DATA LOCAL INFILE :path INTO TABLE syllables
+                FIELDS TERMINATED BY ''
+                (pattern)"
+            );
+
+            $stmt->execute(['path' => $path]);
+        } catch (PDOException $e) {
+            echo $e->getMessage();
+        }
+    }
+
+    public function getAllSyllables(): array
+    {
+        $stmt = $this->connection->query('SELECT * FROM syllables');
+        $unfilteredSyllableArrays = $stmt->fetchAll(PDO::FETCH_ASSOC);
+
+        $filteredSyllables = [];
+
+        foreach ($unfilteredSyllableArrays as $unfilteredSyllables) {
+            $syllable = new Syllable($unfilteredSyllables['id'], $unfilteredSyllables['pattern']);
+            $filteredSyllables[] = $syllable;
```

Comment:
Why not just put to single line?

```php
$filteredSyllables[] = new Syllable($unfilteredSyllables['id'], $unfilteredSyllables['pattern']);
```

__optional__ Also syntax sugar could be applied using `array_map`